### ISSUE TITLE
can not  download apk file properly.

### ISSUE THREAD

**Issue by huqiangjz** [Kooboo/Kooboo#18]
I am a fan of kooboo since codeplex and I am glad to see the project come back again.
when I try to use kooboo , I find it can't let me download apk file properly, but it is ok for jpg , png. pptx etc. 
maybe you should set MIME Type.

![image](https://user-images.githubusercontent.com/49600901/56087049-d482f100-5e95-11e9-86b0-2f29b45e122d.png)

**Comment by koobooteam** [comment]
Yes, agree! You are right. I think this is due to incorrect MIME Type. Will fix it.
